panther_manager/test/plugins/action/test_dock_robot_action_node.cpp

```diff
@@ -0,0 +1,247 @@
+// Copyright 2024 Husarion sp. z o.o.
```

Comment:
How long does it take to execute these tests?

Reply:
14.0s
```bash
colcon test --packages-select panther_manager --ctest-args -R dock 
Starting >>> panther_manager
Finished <<< panther_manager [14.0s]            

Summary: 1 package finished [14.5s]
```

Reply:
That's quite a lot 😨 
Do you spot any chances to limit the execution time?
I guess we're bound by time dependency? Timeouts etc?

Reply:
I changed timeouts:
```
Starting >>> panther_manager
Finished <<< panther_manager [9.69s]  
```
Almost 6 seconds less.